README.md

```diff
@@ -182,7 +182,7 @@ This is the main Robot-Centric Elevation Mapping node. It uses the distance sens
 
 * **`min_update_rate`** (double, default: 2.0)
 
-    The mininum update rate (in Hz) at which the elevation map is updated either from new measurements or the robot pose estimates.
+    The mininum update rate (in Hz) at which the (raw and fused) elevation map is updated either from new measurements or the robot pose estimates. If this rate is 0.0, the (raw) elevation map is only published with new measurements and (fused) elevation map is only published at `fused_map_publishing_rate`.
```

Comment:
Is it correct that if the rate is 0.0, new robot pose estimates do not trigger publishing?

Also, the `triggerFusion` service call triggers publishing, right?

Reply:
The new robot pose estimates do not publish the map.
Yes, `triggerFusion` will trigger the fused map publishing and I think it is already described in the service section.

Reply:
Ok.

Reply:
This is not automatically triggering the map fusion process, unless `isContinouslyFusing_` is true.

Please update with:
"The mininum update rate (in Hz) at which the raw elevation map is updated either from new measurements or the robot pose estimates. If this rate is 0.0, the raw elevation map is only published with new measurements, potentially corrupting data due to when new measurements are not processed for a longer periods."